Automattic/woocommerce-payments
# Updating to modern Woo colours

#### Changes proposed in this Pull Request


Giving these docs the modern Woo purple

Adjusting images and stylesheets to modernize the purple colour used in the documentation powered by Slate.





#### Testing instructions


Make sure the things that are changing reflect the few small color changes i've shown here in the before/after, and make sure i'm not adjusting anything outside of these few color changes. 
![IMG003182](https://github.com/Automattic/woocommerce-payments/assets/1509380/d27c5e6f-3c0f-4e5b-af71-f669a1f76396)




-------------------


**Post merge**
